repos/fdbt-site/src/pages/pointToPointPeriodProduct.tsx

```diff
@@ -120,6 +120,7 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): { props: Po
     const product = getSessionAttribute(ctx.req, POINT_TO_POINT_PRODUCT_ATTRIBUTE);
 
     if (!operatorAttribute?.name) {
+        console.log(operatorAttribute);
```

Comment:
If you want to log you'll need to use `logger.error`.
In this case it may just be easier to change the error message below as you're only changing one thing

Reply:
message updated to 'The Operator Attribute name was not set'